book/theme/index.hbs

```diff
@@ -0,0 +1,384 @@
+<!DOCTYPE HTML>
+<html lang="{{ language }}" class="sidebar-visible no-js {{ default_theme }}">
+    <head>
+        <!-- Book generated using mdBook -->
+        <meta charset="UTF-8">
+        <title>{{ title }}</title>
+        {{#if is_print }}
+        <meta name="robots" content="noindex" />
+        {{/if}}
+        {{#if base_url}}
+        <base href="{{ base_url }}">
+        {{/if}}
+
+        <script async src="https://www.gstatic.com/brandstudio/kato/cookie_choice_component/cookie_consent_bar.v3.js"
+                data-autoload-cookie-consent-bar="true"></script>
```

Comment:
hmm seems we do not need to use cookies - we never track anything!

Reply:
Yeah, don't bother with that — I had to add it to my site because we use Google Analytics there.

Reply:
deleted (didn't check carefully before, thank you for your patience)